docs/root/version_history/current.rst

```diff
@@ -46,6 +46,7 @@ New Features
 ------------
 * http: added :ref:`string_match <envoy_v3_api_field_config.route.v3.HeaderMatcher.string_match>` in the header matcher.
 * http: added support for :ref:`max_requests_per_connection <envoy_v3_api_field_config.core.v3.HttpProtocolOptions.max_requests_per_connection>` for both upstream and downstream connections.
+* http: sanitizing the referer header as documented :ref:`referer_sanitization <config_http_conn_man_headers_referer>` in connection manager. This feature can be temporarily turned off by setting runtime guard ``envoy.reloadable_features.sanitize_http_header_referer`` to false.
```

Comment:
So this will read
http: sanitizing the referer header as documented referer_sanitization
I think it'd make more sense to go with
http: sanitizing the referer header as documented here

If you're still planning on doing an immediate follow-up with more sanitation I' fine with you updating it there.

Reply:
done.